doc/install.rst

```diff
@@ -0,0 +1,29 @@
+Installation
+===============================================
+
+
+To install go to the main folder and do:
+
+```pip install .```
+
+
+Requirements for users
+---------------------------
+
+ - Python 3.6+
+ - python-libsbml>=5.18.0
+ - PyYAML>=5.1
+ - pandas>=1.0.1
+ - petab>=0.1.4
+
+Extras (to run examples):
```

Comment:
if not done yet, I would turn this into an "extras_require".

Reply:
Should already be in there... :)